tests/templates/service/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'ewencp'
```

Comment:
It's handy to update the template so this doesn't crop up all the time (you can stick the copyright notice here)

Pycharm -> preferences ->  editor -> file and code templates -> python script